### PR TITLE
Updated the collaboration section

### PR DESCRIPTION
**Pull request by muffato**
I also moved the team headings to level 3, under level 2 "Teams" so structure things better

### CODE REVIEW

File: markdown/about.md [hunk 1]
```diff
@@ -17,7 +19,12 @@ The [Informatics Infrastructure team](/it) provides support for the production o
 - [Paul Davis](/it#paul-davis)
 - Beth Yates
 
-We welcome external collaboration on our projects - if you'd like to be involved, [drop us a message](https://github.com/sanger-tol/pipelines-website/issues/new?assignees=priyanka-surana%2Cmuffato&labels=connect&projects=&template=contact_us.yaml&title=%5BContact+Us%5D%3A+).
+## Collaborations
+
+We welcome external contributions and collaboration on our projects.
+Iif you'd like to be involved, [drop us a message](https://github.com/sanger-tol/pipelines-website/issues/new?assignees=priyanka-surana%2Cmuffato&labels=connect&projects=&template=contact_us.yaml&title=%5BContact+Us%5D%3A+) or comment directly on an existing GitHub issue.
+
+Please note that we don't have funded internships or student placements at the moment, but are still very happy to help and mentor.
```
Comment (priyanka-surana) [inline markdown/about.md:27]: Do we really need this sentence?

**Review by priyanka-surana — COMMENTED** [review]
We can also add the collaboration message like snippet below the CoC message instead of a dedicated section.

File: markdown/about.md (hunk 1 above)
Comment (muffato) [reply markdown/about.md:27]: I thought it'd be useful to set the expectations, but maybe it just looks bad. Happy to remove it if you think it's not needed

**Comment by muffato** [comment]
> We can also add the collaboration message like snippet below the CoC message instead of a dedicated section. This will also avoid the teams being in an additional category.

Yep, that's a good idea, and would reinforce that the CoC is the basis for the interactions we expect within the community.
I think I'd naturally put it right _before_ the code of conduct, like "please join us but don't forget to follow the CoC", but I'm happy either way.

**Pull request merged**